python/lsst/ts/rubintv/models/models.py

```diff
@@ -150,6 +150,11 @@ class Event:
     filename: str = ""
     ext: str = ""
 
+    def __eq__(self, other: Any) -> bool:
+        if type(other) is not type(self):
+            raise TypeError
+        return self.key == other.key
+
```

Comment:
If I understand this correctly, this is being used by the `max` function to retrieve the event with the biggest key? I've tried to search some references but only find some stackoverflow comments explaining that the `max`function only needs `__lt__` or `__gt__` depending on the case. Is the `__eq__` really needed? Also, can you add some docstrings to both `__eq__` and `__lt__` explaining why they are needed, and why only those two?

Reply:
You're right. The `__eq__` method was left over from a previous implementation. 
I'll remove it and add a docstring to the other dunder.

Reply:
thanks :pray:.